extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -5,6 +5,8 @@
 use Wikia\Logger\WikiaLogger;
 
 class WallMessage {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "WallMessage" has 122 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)